runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
> But the web API spec would tell you how to set it up (e.g. “set a stdin property on your POSTed payload”).

But we're not writing a web-API spec, and we have to specify what a user process can guarantee in an OCI runtime.

> I'd moved it into the command line API and added more information about how the file descriptors get setup. Less information in this file, but more in the spec as a whole.

As someone reading the spec, if I read about the semantics of `/proc/self/fd/...` inside a **CLI** specification, and there's no mention in the main runtime specification file I would assume that they are only defined for `/proc/self/fd`. But they are defined for all containers irrespective of API. So we shouldn't move it IMO.